src/session.rs

```diff
@@ -277,6 +277,15 @@ impl Session {
         )
     }
 
+    pub fn shared_command<'a, S: Into<Cow<'a, str>>>(self: std::sync::Arc<Self>, program: S) -> OwnedCommand<std::sync::Arc<Self>> {
```

Comment:
I think `shared_thread_safe_command` is a better name.
Then we can have `shared_command` for `Rc<...>`

Reply:
Added that change but TBH it feels unwieldingly verbose.

Reply:
Hmmm, maybe arc_command is a better name?